### PR TITLE
Internal change

### PR DESCRIPTION
**Pull request by smolkaj**
Internal change

### CODE REVIEW

**Comment by smolkaj** [comment]
@stefanheule PTAL

File: BUILD.bazel [hunk 1]
```diff
@@ -1,28 +1,14 @@
-# Copyright 2020 Google LLC
```
Comment (stefanheule) [inline BUILD.bazel:1]: This seems like a bug, right? Missing copyright?

File: p4_pdpi/entity_management.cc [hunk 2]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
```
Comment (stefanheule) [inline p4_pdpi/entity_management.cc:1]: I thought these don't work and can't be open-sourced right now?

File: BUILD.bazel (hunk 1 above)
Comment (smolkaj) [reply BUILD.bazel:1]: The third_party team asked me to remove it. Apparently not a thing we do for BUILD files.

File: p4_pdpi/entity_management.cc (hunk 2 above)
Comment (smolkaj) [reply p4_pdpi/entity_management.cc:1]: True, but for simplicity I only removed the BUILD targets and kept the source files.

**Comment by smolkaj** [comment]
@stefanheule, please give me an explicit LGTM once this looks good.

**Review by stefanheule — APPROVED** [review]

**Comment by stefanheule** [comment]
LGTM :)

**Pull request merged**